src/main/flight/mixer.c

```diff
@@ -64,6 +64,7 @@ PG_REGISTER_WITH_RESET_TEMPLATE(mixerConfig_t, mixerConfig, PG_MIXER_CONFIG, 0);
 PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .mixerMode = TARGET_DEFAULT_MIXER,
     .yaw_motors_reversed = false,
+    .ddd_mixergain_halving = true,
```

Comment:
To be consistent with other 3D parameters/variables how about `mix3d_gain_fix`.  Also shouldn't default to true for testing a new experimental fix.  Testers should have to explicitly enable.

Reply:
I'll rename it as suggested.
As it's currently implemented defaulting ddd_mixergain_halving true is what maintains the previous code behavior, it's setting it to off which causes the if statement to fail at line 453 even when 3D mode is enabled, thus removing the halving function from having an effect.

In that vein I can rename it to mix3d_gain_fix, swap it to default false, then change the operator in line 453 from && to !& and achieve the same end result.

Would that be more agreeable? Thanks for the input, it makes sense.

Reply:
Yes, logically the user should be **enabling** the fix that they are testing.  So the OFF state should be default to match the current behavior.

Reply:
I just realised it's ! rather than !& I'm looking for. Not sure if it's hard to read where it sits but should be ok.

Reply:
Even better, the && was supposed to stay and a ! in front of the variable. At least I worked it out before trying to compile or use it!